### PR TITLE
Added AutoHotKey scripts for install/uninstall

### PR DESCRIPTION
**Pull request by mew1033**
Howdy,

I added ahk scripts for both the install and uninstall. It should be completely automated now. I tested it and it seems to be working, although I'm still kinda new at this.

### CODE REVIEW

File: tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+$packageArgs = @{
+  packageName   = $packageName
+  fileType      = 'EXE'
+  url           = $url
+
+  silentArgs = "" # Silent installation discussion: https://veracrypt.codeplex.com/discussions/579539
+
+  softwareName  = 'VeraCrypt*'
+  checksum      = 'aafacca9a600af5b8d66387718c984b8655905f72370bbd772baf90e57e85b7e'
+  checksumType  = 'sha256'
```
Comment (kendaleiv) [inline tools/chocolateyInstall.ps1:16]: Checksum, nice!

File: tools/chocolateyUninstall.ps1 [hunk 2]
```diff
@@ -11,4 +11,23 @@ else {
...
+                           -PassThru
+  Write-Debug "$ahkRun start time:`t$($ahkProc.StartTime.ToShortTimeString())"
+  Write-Debug "$ahkRun process ID:`t$($ahkProc.Id)"
+
+  Uninstall-ChocolateyPackage $packageName $fileType $args $setupExePath
+
+  Remove-Item "$ahkRun" -Force
+} catch {
+  throw $_.Exception
+}
```
Comment (kendaleiv) [inline tools/chocolateyUninstall.ps1]: Could you remove the try/catch here?

File: veracrypt.nuspec [hunk 3]
```diff
@@ -6,13 +6,16 @@
...
     <authors>IDRIX</authors>
-    <owners>kendaleiv</owners>
+    <owners>kendaleiv, mew1033</owners>
     <summary>VeraCrypt</summary>
     <description>VeraCrypt is a free disk encryption software brought to you by IDRIX (https://www.idrix.fr) and that is based on TrueCrypt.
 
 - Installation and uninstallation are not silent / unattended. See https://veracrypt.codeplex.com/discussions/579539</description>
     <projectUrl>https://veracrypt.codeplex.com/</projectUrl>
     <iconUrl>https://cdn.rawgit.com/kendaleiv/chocolatey-veracrypt/master/veraCrypt.png</iconUrl>
+	<dependencies>
```
Comment (kendaleiv) [inline veracrypt.nuspec]: Would you mind changing the tab to spaces here? :smiley:

File: tools/chocolateyUninstall.ps1 (hunk 2 above)
Comment (mew1033) [inline tools/chocolateyUninstall.ps1]: Done.

File: veracrypt.nuspec (hunk 3 above)
Comment (mew1033) [inline veracrypt.nuspec]: Done.

**Comment by mew1033** [comment]
Made the requested changes.

**Comment by kendaleiv** [comment]
Works great on fresh Windows Server 2012 R2 Datacenter Azure VM!

I had to manually install the dependency since I was targeting the package locally, but I'm hoping that won't be an issue.

**Comment by kendaleiv** [comment]
@mew1033 FYI submitted a new nupkg to https://chocolatey.org/ with this pull request implemented, plus pull request #2 to update the description -- since it's now an unattended install.

Thanks so much!

**Comment by kendaleiv** [comment]
@mew1033 Since I mentioned in the previous comment about what was submitted, I thought I'd also mention I just submitted again with the changes in PR #4 implemented.

**Comment by mew1033** [comment]
@kendaleiv Sweet, glad it all got merged! Sorry I forgot to update the description. :-)

**Comment by kendaleiv** [comment]
@mew1033 No worries, you did the hard work on this one. :smiley:

**Pull request merged**